src/DateMonth.js

```diff
@@ -100,7 +102,14 @@ export default class DateMonth extends Component {
     const change = event => {
       if (this._input.checkValidity()) {
         const text = event.target.value.trim();
-        const date = fecha.parse(text, 'MMM YYYY');
+        var tmp_date;
+        if (text.match(MMYY_PATTERN)) {
+          tmp_date = fecha.parse(text, 'MM/YY');
+        }
+        else {
+          tmp_date = fecha.parse(text, 'MMM YYYY');
+        }
+        const date = tmp_date;
```

Comment:
I'd probably recommend using the ternary operator here to avoid temp variables:

    const date = text.match(MMYY_PATTERN) ?
              fecha.parse(text, 'M/YY') :
              fecha.parse(text, 'MMM YYYY');